.devcontainer/genesis-example.json

```diff
@@ -10,6 +10,11 @@
     "petersburgBlock": 0,
     "istanbulBlock": 0,
     "muirGlacierBlock": 0,
+    "durangoTimestamp": 0,
```

Comment:
we can omit durangoTimestamp as it is activated by default

Reply:
Removed